WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModel.kt

```diff
@@ -472,6 +474,19 @@ class ReaderPostDetailViewModel @Inject constructor(
         )
     }
 
+    private fun onCommentsClicked() {
+        post?.let {
+            launch {
+                readerPostCardActionsHandler.onAction(
+                    it,
```

Comment:
Nitpick: since we're already naming part of the arguments, would it make sense to name **all** of them?
e.g.

```
                readerPostCardActionsHandler.onAction(
                    post = it,
                    type = COMMENTS,
                    isBookmarkList = false,
                    source = ReaderTracker.SOURCE_POST_DETAIL,
                )
```